packages/docusaurus-theme-classic/src/theme/DocSidebar/index.js

```diff
@@ -36,6 +36,31 @@ function DocSidebarItem({item, onItemClick, collapsible, ...props}) {
     setCollapsed((state) => !state);
   });
 
+  let activePageRelativeUrl = '';
+
+  // Because this is built on the server
+  // we need to check if window is available
+  if (typeof window !== 'undefined') {
```

Comment:
@jsjoeio you can use an API  targeted for this purpose, see details here -> https://v2.docusaurus.io/docs/docusaurus-core/#executionenvironment

Reply:
Wow. I didn't know that was a thing. TIL. Thanks for the tip!